README.md

```diff
@@ -0,0 +1,28 @@
+# LSST code templates
+
+This repository provides standardized templates for LSST software projects.
+Refer to these templates when creating new projects (such as stack packages) and when adding new files to existing repositories.
+We're continuously updating these templates with our adopted standards and best practices.
+
+Two classes of templates are available in separate directories:
+
+
+1. `templates/`.
```

Comment:
It's a bit confusing to me to have a repository called `templates` with a `templates` directory and a `projects` directory.  It makes me wonder why there are two: i.e. it is not immediately clear that `projects` contains templates of a different kind.

I think it might help to call the directory `file_templates` and maybe even `projects` --> `project_templates`.